scene/debugger/scene_debugger.cpp

```diff
@@ -1089,4 +1185,943 @@ void LiveEditor::_reparent_node_func(const NodePath &p_at, const NodePath &p_new
+	mat->set_flag(StandardMaterial3D::FLAG_DISABLE_FOG, true);
+	// In the original Node3DEditor, this value would be fetched from the "editors/3d/selection_box_color" editor property,
+	// but since this is not accessible from here, we will just use the default value.
+	const Color selection_color_3d = Color(1, 0.5, 0);
+	mat->set_albedo(selection_color_3d);
+	mat->set_transparency(StandardMaterial3D::TRANSPARENCY_ALPHA);
+	st->set_material(mat);
+	sbox_3d_mesh = st->commit();
+
+	Ref<StandardMaterial3D> mat_xray = memnew(StandardMaterial3D);
+	mat_xray->set_shading_mode(StandardMaterial3D::SHADING_MODE_UNSHADED);
+	mat_xray->set_flag(StandardMaterial3D::FLAG_DISABLE_FOG, true);
+	mat_xray->set_flag(StandardMaterial3D::FLAG_DISABLE_DEPTH_TEST, true);
+	mat_xray->set_albedo(selection_color_3d * Color(1, 1, 1, 0.15));
+	mat_xray->set_transparency(StandardMaterial3D::TRANSPARENCY_ALPHA);
+	st_xray->set_material(mat_xray);
+	sbox_3d_mesh_xray = st_xray->commit();
+#endif // _3D_DISABLED
+
+	SceneTree::get_singleton()->connect("process_frame", callable_mp(this, &RuntimeNodeSelect::_process_frame));
+	SceneTree::get_singleton()->connect("physics_frame", callable_mp(this, &RuntimeNodeSelect::_physics_frame));
+
+	// This function will be called before the root enters the tree at first when the Game view is passing its settings to
+	// the debugger, so queue the update for after it enters.
+	root->connect(SceneStringName(tree_entered), callable_mp(this, &RuntimeNodeSelect::_update_input_state), Object::CONNECT_ONE_SHOT);
+}
+
+void RuntimeNodeSelect::_node_set_type(NodeType p_type) {
+	node_select_type = p_type;
+	_update_input_state();
+}
+
+void RuntimeNodeSelect::_select_set_mode(SelectMode p_mode) {
+	node_select_mode = p_mode;
+}
+
+void RuntimeNodeSelect::_set_camera_override_enabled(bool p_enabled) {
+	camera_override = p_enabled;
+
+	if (p_enabled) {
+		_update_view_2d();
+	}
+
+#ifndef _3D_DISABLED
+	if (camera_first_override) {
+		_reset_camera_2d();
+		_reset_camera_3d();
+
+		camera_first_override = false;
+	} else if (p_enabled) {
+		_update_view_2d();
+
+		SceneTree::get_singleton()->get_root()->set_camera_3d_override_transform(_get_cursor_transform());
+		SceneTree::get_singleton()->get_root()->set_camera_3d_override_perspective(CAMERA_BASE_FOV * cursor.fov_scale, CAMERA_ZNEAR, CAMERA_ZFAR);
+	}
+#endif // _3D_DISABLED
+}
+
+void RuntimeNodeSelect::_root_window_input(const Ref<InputEvent> &p_event) {
+	Window *root = SceneTree::get_singleton()->get_root();
+	if (node_select_type == NODE_TYPE_NONE || selection_list->is_visible()) {
+		// Workaround for platforms that don't allow subwindows.
+		if (selection_list->is_visible() && selection_list->is_embedded()) {
+			root->set_disable_input_override(false);
+			selection_list->push_input(p_event);
+			// For whatever reason, using `callable_mp` here makes so that the hovered item isn't highlighted.
+			root->call_deferred(SNAME("set_disable_input_override"), true);
```

Comment:
@YeldhamDev `set_disable_input_override` is not registered in the method map, which causes this to be a no-op.
Adding this to `Viewport::_bind_methods()` should fix the issue:
```
ClassDB::bind_method(D_METHOD("set_disable_input_override", "disable"), &Viewport::set_disable_input_override);
```

Another approach if you don't want to expose this method would be to reset input override when the popup is dismissed.

Reply:
You could also use `callable_mp()` rather than binding it. It'd be something like:

```c++
callable_mp(root, &Viewport::set_disable_input_override).call_deferred(true);
```

Reply:
Alright, I resorted to use `callable_mp` anyways. The hovered item doesn't get highlighted as per the comment, but selecting works.